### ISSUE TITLE
ensure correct working order of website and expected styles (e.g. shelf and scrolling, docs, markdown plugins, etc)

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#432]
## Type of Change
- [ ] New Feature Request
- [x] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
As part of #355 and #418 , there were some warts left into the website, like:
1. [ ] click areas for the shelf loading and chevron expansion
1. [ ] make sure heading scrolling works
1. [ ] make sure custom markdown plugins from _greenwood.config.js_ are working
1. [ ] Noticed `#### Examples` headings are showing up in side bars - https://github.com/ProjectEvergreen/greenwood/pull/452#issuecomment-750407404
1. [ ] generally make sure everything matches www.greenwoodjs.io as it did before

## Details
Basically make sure the website is still working / looking as expected, and fix what isn't working.  Should save this for the end.

**Comment by thescientist13** [comment]
Noticed the Plugins table styles are off a little bit.
<img width="1179" alt="Screen Shot 2020-12-30 at 11 19 43 AM" src="https://user-images.githubusercontent.com/895923/103366579-033d6280-4a91-11eb-8971-9ad0ff3fb8c1.png">

I think in my effort to keep the styles still working in an un-nested manner until PostCSS could land, I may have changed some of the site structure.  Maybe good if someone could just double check that because once #426 lands, everything should be back to normal from a tooling perspective.

**Comment by thescientist13** [comment]
>  Noticed #### Examples headings are showing up in side bars - #452 (comment)

I think this may be caused by our commented out code still being picked up by the auto headings markdown plugin?
